front/hooks/useHandleMembersRoleChange.ts

```diff
@@ -0,0 +1,84 @@
+import type {
+  LightWorkspaceType,
+  RoleType,
+  UserTypeWithWorkspaces,
+} from "@dust-tt/types";
+import { useCallback, useContext } from "react";
+import { useSWRConfig } from "swr";
+
+import { SendNotificationsContext } from "@app/components/sparkle/Notification";
+
+type HandleMembersRoleChangeParams = {
+  members: UserTypeWithWorkspaces[];
+  role: RoleType;
+};
+
+export function useHandleMembersRoleChange({
```

Comment:
Thanks for creating the hook :)

Couple of nits:
- `useChangeMembersRoles()` instead ?
- Put the mutation(s) directly there when success ?
- Move at the end of "swr/memberships" (to keep CRUD actions together ?)